### DISCUSSION TITLE
Unrecognized Resource API Version

### DISCUSSION THREAD

**Discussion by Chambras** [microsoft/vscode-azurearmtools#1532] [Q&A] [answered]
The wiki talks about this error [Unrecognized Resource API Version](https://github.com/microsoft/vscode-azurearmtools/wiki/Unrecognized-Resource-API-Version) and it mentions there is a page with a list of all the resource providers that are supported, and link goes to https://github.com/Azure/azure-resource-manager-schemas/blob/master/generator/whitelist.ts which is broken. What us the updated page that has this list?

**Comment by StephenWeatherford** [comment, marked as answer]
Thanks for letting us know.  Looks like this is the new version: https://github.com/Azure/azure-resource-manager-schemas/blob/main/generator/resources.json.  I've updated the wiki.
